cmd/kubeadm/app/util/config/initconfiguration_test.go

```diff
@@ -34,16 +34,16 @@ import (
 )
 
 const (
-	master_v1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
-	master_v1alpha3YAMLNonLinux  = "testdata/conversion/master/v1alpha3_non_linux.yaml"
-	master_v1beta1YAML           = "testdata/conversion/master/v1beta1.yaml"
-	master_v1beta1YAMLNonLinux   = "testdata/conversion/master/v1beta1_non_linux.yaml"
-	master_internalYAML          = "testdata/conversion/master/internal.yaml"
-	master_internalYAMLNonLinux  = "testdata/conversion/master/internal_non_linux.yaml"
-	master_incompleteYAML        = "testdata/defaulting/master/incomplete.yaml"
-	master_defaultedYAML         = "testdata/defaulting/master/defaulted.yaml"
-	master_defaultedYAMLNonLinux = "testdata/defaulting/master/defaulted_non_linux.yaml"
-	master_invalidYAML           = "testdata/validation/invalid_mastercfg.yaml"
+	masterV1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
```

Comment:
in a follow up PR we should rename `master` and the folder names to to `controlplane` or something similar.

Reply:
@neolit123 Thanks! I will pay attention to the PRs in the future.